base/save.go

```diff
@@ -17,20 +17,11 @@ import (
 	"github.com/qri-io/qri/startf"
 )
 
-// SaveDatasetSwitches provides toggleable flags to SaveDataset that control
-// save behaviour
-type SaveDatasetSwitches struct {
-	Replace             bool
-	DryRun              bool
-	Pin                 bool
-	ConvertFormatToPrev bool
-	Force               bool
-	ShouldRender        bool
-	NewName             bool
-}
+// SaveSwitches is an alias for the switches that control how saves happen
+type SaveSwitches = dsfs.SaveSwitches
```

Comment:
I was going to argue against this alias but I'm delighted to see none of this is exposed at the lib level, so I think it's fine considering it's "internal"